src/commands/commands.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-restricted-globals */
 /* global Office */
```

Comment:
Is the global comment actually needed? The types package https://www.npmjs.com/package/@types/office-js, that we already use, declares `Office` as a global variable.

Reply:
You propose adding it with `JSX` in `.eslintrc.json` -> remove all `/* global Office */`?

Reply:
I thought the ts compiler would take care of this, so that one can simply remove the global comment.

Reply:
We are 100% TS and should [turn off `no-undef`](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md#i-get-errors-from-the-no-undef-rule-about-global-variables-not-being-defined-even-though-there-are-no-typescript-errors), in which case we can likely remove JSX from Globals (in `eslintrc.json`) as well.

Reply:
@mohit038 this was what I meant with turning off `no-undef` (hopefully) instead of declaring `JSX` and `Office` as global variables.